README.md

```diff
@@ -16,14 +16,13 @@ This action gets a list of GitHub deployments and statuses created by [im-open/c
 - [Code of Conduct](#code-of-conduct)
 - [License](#license)
 
-When the action runs it will add a deployment and deployment status record to the repo.  The GitHub Deployment will also have a special task d
 
 ## Inputs
 
 | Parameter     | Is Required | Description                                                                                                                                                                                                                       |
 | ------------- | ----------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `token`       | true        | A token with `repo_deployment` permissions to create and update issues, workflows using this action should be granted `permissions` of `deployments: write`                                                                       |
-| `environment` | true        | The name of a GitHub environment the release was deployed to, i.e. [Dev\|QA\|Stage\|Demo\|UAT\|Prod]. It will be set in the standard `environment` property of the deployment object.                                             |
+| `token`       | true        | A token with `deployments:read` and `contents:read` to get the deployments and release reference data.                                                                                                                            |
+| `environment` | true        | The name of a GitHub environment the release was deployed to, i.e. [Dev\|QA\|Stage\|Demo\|UAT\|Prod]. It ill be used to filter deployment objects.                                                                                |
```

Comment:
I can't leave a recommendation, I'm guessing because we aren't codeowners at the moment, but 

> It ill be used ...

needs to change to

> It will be used ...

Reply:
👍🏽